src/components/Image/Image.tsx

```diff
@@ -0,0 +1,14 @@
+import React, { FC } from "react";
+import styles from "./Image.css";
+import cx from "classnames";
+
+interface Props {
+  src: string;
+  alt: string;
```

Comment:
Ønsker også at Image skal ha onClick metode som prop sånn at vi kan sende inn det.

Vi må også kunne definere størrelsen på bildet utenfra tenker jeg?

Reply:
Bilde-komponenten her e hovedsakelig tenkt for de som skal brukes på Card, ikke vanlig bildefremvisning, så tenker at da trenger man ikke onClick. Størrelsen på bilde settes dermed også ved at den er relativ til diven rundt.